src/internal/AnyCatcher.ts

```diff
@@ -0,0 +1,3 @@
+export interface AnyCatcher {
```

Comment:
LOLOLOLOLOLOLOLOL 🤣 🎉

Reply:
suggestion: This could be something like this, instead:

```ts
declare const anyCatcherSymbol: unique symbol;
type AnyCatcher = typeof anyCatcherSymbol;
```

[TypeScript Playground demo](https://www.typescriptlang.org/play?#code/CYUwxgNghgTiAEYD2A7AzgF3lFBPAwlBmABYgwDKuAtgEZIQBc8ArigJYCOLCaN9EANwBYAFAZcABwQBBPIWJkY8ALzwJ0pADNs8oqXJU6DEaLGhIsBFrZgM7VPC0AeACrwQADwwgUwNPByBPpKAHwAFLAA5syuAJTMbADWKEgA7iimFtBwTrb2ji6uEdGxCfCupmLI6FhQqk7hAERQTdgBOLhxgvAA9L2sKCnpKNWomPC0DVrNtE3dfQNNc0A)

Reply:
Yeah, that's what I was saying on Slack, but it's SO BORING. Haha. Okay fine.